internal/schemautil/wait.go

```diff
@@ -246,54 +242,50 @@ func grafanaReady(service *aiven.Service) bool {
 	return true
 }
 
-func backupsReady(service *aiven.Service) bool {
-	if service.Type != "pg" && service.Type != "elasticsearch" &&
-		service.Type != "redis" && service.Type != "influxdb" {
-		return true
-	}
-
-	if service.Type == "redis" && service.UserConfig["redis_persistence"] == "off" {
+func backupsReady(s *service.ServiceGetOut) bool {
+	switch s.ServiceType {
+	case ServiceTypePG, ServiceTypeInfluxDB, ServiceTypeRedis, ServiceTypeDragonfly:
+		// See https://github.com/aiven/terraform-provider-aiven/issues/756
+		switch "off" {
+		case s.UserConfig["redis_persistence"], s.UserConfig["dragonfly_persistence"]:
+			return true
+		}
+	default:
 		return true
 	}
 
-	// no backups for read replicas type of service
-	for _, i := range service.Integrations {
-		if i.IntegrationType == "read_replica" && *i.DestinationService == service.Name {
+	// No backups for read replicas type of service
+	// See https://github.com/aiven/terraform-provider-aiven/pull/172
+	for _, i := range s.ServiceIntegrations {
+		if i.IntegrationType == "read_replica" && *i.DestService == s.ServiceName {
 			return true
 		}
 	}
 
-	return len(service.Backups) > 0
+	return len(s.Backups) > 0
```

Comment:
I wonder if I should add OpenSearch here because it used to have ES. But we have been running this code for years with no issues.